dhall-lsp-server/src/Dhall/LSP/State.hs

```diff
@@ -40,21 +41,19 @@ data Severity = Error
               -- ^ Log message, not displayed by default.
 
 data ServerConfig = ServerConfig
-  { asciiOnly :: Bool
-  -- ^ Use ASCII symbols rather than fancy unicode when formatting and linting
-  -- code.
+  { chosenCharacterSet :: Maybe CharacterSet
   } deriving Show
```

Comment:
@PanAeon Do you know if I need to modify something else to make this new config format work? Maybe somewhere in https://github.com/PanAeon/vscode-dhall-lsp-server ? I haven't seen anything that sets the LSP config there though.

Reply:
It looks like it works just fine without modifying the VS code extension, and this config can still be set from `settings.json` manually. I might try to make this new option accessible from the extension's UI, but it's not necessary for now.